typings/objection/index.d.ts

```diff
@@ -1099,6 +1099,7 @@ declare namespace Objection {
     mergeContext: ContextMethod<this>;
 
     modify: ModifyMethod<this>;
+    modifiers: ModifyMethod<this>;
```

Comment:
This type is not correct. Modifiers takes an object full of modifiers. Something like

```js
interface ModifiersMethod<QB extends AnyQueryBuilder> {
  (modifiers: Modifiers): QB
}
```

Reply:
Thank you! I was gonna ask about it. 

On a side note, I really like that API :+1: